compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -227,7 +227,7 @@ basicTests mbScenarioService = Tasty.testGroup "Basic tests"
             setFilesOfInterest [a]
             expectWarning (a,0,25) "The import of ‘B’ is redundant"
             setBufferModified a "???"
-            expectOneError (a,0,0) "Missing daml version"
```

Comment:
I must be missing something: How was there a "Missing daml version" error in the past?

Reply:
That is exactly the thing I missed. Thanks for pointing me at it.